docs/getting-started.md

```diff
@@ -26,3 +26,66 @@ sudo daed [-h,--help]
 If everything goes well, open your browser and navigate to `http://localhost:2023`
 
 Happy Hacking!
```

Comment:
Please place this line at the end of the file.

Reply:
see #246